splink/files/chart_defs/parameter_estimate_comparisons.json

```diff
@@ -2,14 +2,14 @@
   "$schema": "https://vega.github.io/schema/vega-lite/v5.2.0.json",
   "title": {
     "text": "Comparison of parameter estimates across training sessions",
-    "subtitle": "Use mousewheeel to zoom"
+    "subtitle": "Use mousewheel to zoom"
```

Comment:
Not an issue, but why is this highlighted as changed? 🤨

Reply:
Cos there was an extra "e"

Reply:
Ah, I feel like a genius now.